lib/view_model/wallet_keys_view_model.dart

```diff
@@ -102,6 +102,12 @@ abstract class WalletKeysViewModelBase with Store {
             title: S.current.wallet_recovery_height,
             value: restoreHeight.toString()));
       }
+      final passphrase = (_appStore.wallet as MoneroWalletBase).passphrase();
+      if (passphrase != null) {
+        items.add(StandartListItem(
+            title: S.current.passphrase,
+            value: passphrase.toString()));
+      }
```

Comment:
we said that we won't show the passphrase in the seeds/keys screen

Reply:
it will be very easy for people to lose their seed then..